config/main.toml

```
@@ -49,9 +49,11 @@ config_version = 1
         _preset = "state_monitor.outputs.plugin"
         power_delim = 20  # delimiter between normal power and low power
         low_space = 100  # if # of GB is less than this, generate a notification
-        method = "slack"  # the method how we're going to send notifications
+        method = "google chat"  # the method how we're going to send notifications
         slack_channel = "testing"  # the channel in slack the notifications will be posted
         slack_key_file = "/home/pi/.slack"  # The path the slack key file
+	    google_chat_channel = "status"
+	    google_chat_key_file = "/home/pi/.googlechat"
```

```suggestion
        channel = "status"  # The chat channel to which notifications will be posted
        key_file = "/home/pi/.googlechat"  # The path to the key file
```

Instead of having duplicate params that only apply to one `method`, IMO would be simpler to just have one set of params, since only one will ever be used at a time. If you do need to store the config for multiple `methods` and toggle between them in the config, you could use separate steps either here or in a `.preset` file, or simply comment out the old values.

Also, fixed something weird going on with the indents here (probably tabs vs spaces, I'd bet).